.github/workflows/continuous-integration.yml

```diff
@@ -77,6 +77,14 @@ jobs:
           arguments: coveralls
         continue-on-error: true
         if: runner.os == 'Linux' && matrix.java == '11' && matrix.epVersion == '2.19.1' && github.repository == 'uber/NullAway'
+      - name: Test publishToMavenLocal flow
+        env:
+          ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
+          ORG_GRADLE_PROJECT_VERSION_NAME: '0.0.0.1-LOCAL'
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: publishToMavenLocal -x signMavenPublication -x signShadowPublication
```

Comment:
1. I go back and forth about whether to include `clean` here. On the one hand, that's what we do locally. On the other, it's a waste of CI resources to build twice and we don't really mind if this is more brittle than locally, unless it's super noisy.
2. Not sure if it's worth setting up dummy signing credentials. Right now, the lack of them means we aren't testing `signMavenPublication` or `signShadowPublication`, and locally, before the revert, I did see errors related to those when running `publish` (but a different error shows up anyways with the command above, so we would have caught this PR either way)

Reply:
1. I think it's fine not to run `clean`.
2. Don't we have credentials set up somewhere to push snapshot builds?  Or are those unsigned?

Reply:
Snapshot builds are unsigned, which is also why we didn't see any issue with the upgrade, generally snapshot builds are different. This check also didn't trigger the error until I added `ORG_GRADLE_PROJECT_VERSION_NAME: '0.0.0.1-LOCAL'` making a (meaninglessly numbered) non-SNAPSHOT version.